sdk/python/feast/infra/registry/registry.py

```diff
@@ -178,6 +178,10 @@ def __new__(
             from feast.infra.registry.snowflake import SnowflakeRegistry
 
             return SnowflakeRegistry(registry_config, project, repo_path)
+        elif registry_config and registry_config.registry_type == "remote":
```

Comment:
from this point I felt like the `Registry` is more like a common class instead of a file registry?

Reply:
yeah, confused me a lot as well. It probably used to be the common one, but then another base class was added and now it's a bit jumbled.